src/leappto/cli.py

```diff
@@ -66,6 +67,9 @@ def _add_identity_options(cli_cmd, context=''):
         cli_cmd.add_argument('--identity', default=None, help='Path to private SSH key')
         cli_cmd.add_argument('--ask-pass', '-k', action='store_true', help='Ask for SSH password')
         cli_cmd.add_argument('--user', '-u', default=None, help='Connect as this user')
+    cli_cmd.add_argument('--unsafe-disable-host-check', default=False, action='store_true',
```

Comment:
this is really annoying to type, just make it no host check, on the other side we don't really need this we can use SSH config like other software does.